### DISCUSSION TITLE
How to put an image in a menu like another link of this menu

### DISCUSSION THREAD

**Discussion by Cod313** [community/community#21473] [Other Feature Feedback, Questions, & Ideas]
<p>Hello,<br>
I need your help,<br>
I create a website with html and CSS and I’ve a photo (my website’s logo) that I will put I my menu like a link of the menu. But I try to move the photo and I can’t do it.<br>
Here my HTML and CSS code of this menu :</p>
<pre><code class="lang-auto">
&lt;ul id="menu"&gt;
  &lt;li&gt;
    &lt;a href="home.html"&gt;&lt;img src="logo.png" class="button" alt="Logo" title="Go home" width="30px" height="50px" /&gt;&lt;/a&gt;
  &lt;/li&gt;
  &lt;li&gt; &lt;a href="visit.html" class="bouton" title="Visit"&gt;Visit&lt;/a&gt; &lt;/li&gt;
  &lt;li&gt; &lt;a href="contact.html" class="bouton" title="Contact"&gt;Contact&lt;/a&gt; &lt;/li&gt;
  &lt;li&gt; &lt;a href="tourism.html" class="bouton"&gt;How make here ?&lt;/a&gt; &lt;/li&gt;
  &lt;li&gt; &lt;a href="reservation.html" class="bouton"&gt;Reservations&lt;/a&gt; &lt;/li&gt;
&lt;/ul&gt;

</code></pre>
<pre><code class="lang-auto">
header {
  background-color: dark;
  border-radius: 10px;
}

#menu li {
  display: inline-block;
}

#menu .bouton {
  display: block;
  width: 8em;
  padding: 5px 0;
  text-align: center;
  border: 1px solid black;
  background-color: white;
  color: black;
  text-decoration: none;
  margin: 50px;
  border-radius: 5px;
}

#menu .bouton:hover {
  background-color: black;
  color: white;
  border-color: white;
}

#menu {
  text-align: center;
}

li {
  font-family: Impact, sans-serif;
  font-size: 1.25em;
}

</code></pre>
<p>Can you help me ?<br>
Thank you very much</p>

**Comment by SethClydesdale** [comment]
<p>Hello <a class="mention" href="/cod313">@Cod313</a>,</p>
<p>Could you please provide the following information?</p>
<ul>
<li>If possible: a link to the website where the issue occurs.</li>
<li>The code in question you’re having trouble with. (if it’s in a repository, please link it here)</li>
<li>Where <em>specifically</em> in this menu do you want the “photo” moved to?</li>
</ul>
<p>It’s imperative that you provide us with as much information as possible so that we may assist you.</p>

**Comment by Cod313** [comment]
<p>Thank you, I’ve added my code</p>

**Comment by SethClydesdale** [comment]
<p>Thank you for providing some additional information. If I may ask, what issue is occurring with the image? Looking at the markup it appears to be fine, so I’m assuming the issue is with the display – the CSS?</p>
<p>Perhaps this will work?</p>
<pre><code class="lang-auto">&lt;ul id="menu"&gt;
  &lt;li&gt;
    &lt;a href="home.html" class="bouton"&gt;&lt;img src="logo.png" class="logo-bouton" alt="Logo" title="Go home" /&gt;&lt;/a&gt;
  &lt;/li&gt;
  &lt;li&gt; &lt;a href="visit.html" class="bouton" title="Visit"&gt;Visit&lt;/a&gt; &lt;/li&gt;
  &lt;li&gt; &lt;a href="contact.html" class="bouton" title="Contact"&gt;Contact&lt;/a&gt; &lt;/li&gt;
  &lt;li&gt; &lt;a href="tourism.html" class="bouton"&gt;How make here ?&lt;/a&gt; &lt;/li&gt;
  &lt;li&gt; &lt;a href="reservation.html" class="bouton"&gt;Reservations&lt;/a&gt; &lt;/li&gt;
&lt;/ul&gt;
</code></pre>
<pre><code class="lang-css">header {
  background-color: dark;
  border-radius: 10px;
}

#menu li {
  display: inline-block;
}

#menu .bouton {
  display: block;
  width: 8em;
  padding: 5px 0;
  text-align: center;
  border: 1px solid black;
  background-color: white;
  color: black;
  text-decoration: none;
  margin: 50px;
  border-radius: 5px;
  box-sizing:border-box;
  height:36px;
  white-space:nowrap;
  overflow:hidden;
}

#menu .logo-bouton {
  height:100%;
}

#menu .bouton:hover {
  background-color: black;
  color: white;
  border-color: white;
}

#menu {
  text-align: center;
}

li {
  font-family: Impact, sans-serif;
  font-size: 1.25em;
}
</code></pre>

**Comment by Cod313** [comment]
<p>Thank you very much <a class="mention" href="/SethClydesdale">@SethClydesdale</a><br>
I’m very grateful to your help :blush:<br>
I’ve tried it an it’s occur that I have a button like the others but my photo (who’s in the same folder) doesn’t appear.<br>
I’m looking to find a response to this problem.</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hmm… that may prove to be a bit difficult to solve without having the live page to debug. Only suggestion I can offer is to check your developer console:</p>
<ol>
<li>Open the devtools (F12; CTRL+SHIFT+I; Right click &gt; inspect)</li>
<li>Move to the elements tab.</li>
<li>Click the button with the cursor and a box OR click the page and press CTRL+SHIFT+C.</li>
<li>Select the image.</li>
</ol>
<p>If done correctly, you can hover over the <strong>src</strong> attribute to see where the resource is coming from.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6b9f3fc5c95d33e7a383d0a3183af083f786cd12.png" data-download-href="/uploads/short-url/fm4hiqgBZLy37OsALmNnR2cnnhg.png?dl=1" title="src" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10122549/181064902-ca59ffea-c7bc-4974-994e-f9d67d79500f.png" alt="src" data-base62-sha1="fm4hiqgBZLy37OsALmNnR2cnnhg" width="690" height="294" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6b9f3fc5c95d33e7a383d0a3183af083f786cd12_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">src</span><span class="informations">805×343 13.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If it’s pulling the resource from the wrong destination, you’ll need to edit the src attribute so it’s referencing the correct location. Most commonly, you’ll need to:</p>
<ul>
<li>reference the sub-directory <code>src="images/logo.png"</code>
</li>
<li>a parent directory <code>src="../logo.png"</code>
</li>
<li>or both <code>src="../images/logo.png"</code>
</li>
</ul>
<p>It depends on what directory the image and page are located in.</p>
<p>Lastly, if the above doesn’t work, check your console for any errors as it’ll let you know the status of any files which aren’t working properly, and may offer some hints on how to solve the issue.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/6113088b36d6d29fb0680b3d886b2209af09db17.png" data-download-href="/uploads/short-url/dQL5NdHCguJlsUv34SZVKj254XB.png?dl=1" title="status" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/10122549/181064903-05efebc6-3ac7-46df-896d-01798ab1d5af.png" alt="status" data-base62-sha1="dQL5NdHCguJlsUv34SZVKj254XB" width="690" height="79" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/6113088b36d6d29fb0680b3d886b2209af09db17_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">status</span><span class="informations">1135×130 7.84 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Cod313** [comment]
<p>Thanks <a class="mention" href="/SethClydesdale">@SethClydesdale</a> for your help.<br>
I’ve made a break sometimes and today I read your post that resolved my problem.<br>
Thanks you very much for all</p>
